README.md

```diff
@@ -79,6 +79,65 @@ cd corral
 
 You can make a debug build with `.\make.ps1 build -Config Debug`
 
+## Getting started using Corral
+After instllation, make sure Corral's current path is also set in your PATH environment variable. This will make it easier by not entering the full path each time when you use one of Corral commands in your projects.
+
+Follow these steps to create your first project using Corral.
+
+1. Create an empty folder and switch to this directory. This will be our example project to use Corral
+    ```bash
+    mkdir myproject
+    cd myproject
+    ```
+
+2. Initialize Corral. It will create `corral.json` and `lock.json` files. At this moment they won't have much information since we haven't added any dependency yet.
```

Comment:
here you referring to the audience as "we" and anove you are using "your" as in "your first project using corral"

The rest of the document is using you/your for the audience in such examples. 

Any "we" should be updated to use you/your to match the rest of the document. (and any others in this addition that might also be "we").